spacelift/resource_worker_pool.go

```diff
@@ -31,6 +31,12 @@ func resourceWorkerPool() *schema.Resource {
 				Description: "name of the worker pool",
 				Required:    true,
 			},
+			"csr": &schema.Schema{
```

Comment:
I was under the impression that we'd generate the private key and the CSR and export the private key as computed value. Or did I misunderstand @cube2222 ?